Fika.Core/UI/Custom/MatchMakerUIScript.cs

```diff
@@ -377,36 +331,6 @@ private void RefreshUI()
                     continue;
                 }
 
-                if (RaidSettings.Side != entry.Side)
-                {
-                    string errorText = "Cannot join a raid that is on another map.";
-                    if (RaidSettings.Side == ESideType.Pmc)
```

Comment:
This should possibly be re-added as well as it should be an option whether people want to allow it or not.
Also, how does it know whether to start the raid in the group as a SCAV or PMC?